libs/message-coder/src/uint32_coder.ts

```diff
@@ -11,11 +11,25 @@ import {
 } from './types';
 import { UintCoder } from './uint_coder';
 
+interface Uint32CoderOptions {
+  littleEndian: boolean;
+}
+
 /**
  * Coder for a uint32, aka a 32-bit unsigned integer. Uses little-endian byte
  * order.
  */
 export class Uint32Coder extends UintCoder {
+  private readonly littleEndian: boolean;
+
+  constructor(
+    enumeration?: unknown,
+    options: Uint32CoderOptions = { littleEndian: true }
+  ) {
+    super(enumeration);
+    this.littleEndian = options?.littleEndian;
+  }
```

Comment:
Wasn't sure if we should put this in `UintCoder`, but since we only need big endian support in Uint16 and Uint32 right now, it felt like overkill to add it at the next level up.